docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Sourcing from Seams-CMS
+---
+
+## Overview
+
+[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to easily manage content and data with either simple or complex structures and supports localization.
+
+Creating a Seams-CMS account is free for hobby or small sites. For this tutorial, we will be using the demo workspace that is automatically generated when you create a new account. Make sure you add an API key to this workspace in the API key settings. We will be needing your workspace ID and the (read-only) API key shortly.
```

Comment:
```suggestion
Creating a Seams-CMS account is free for hobby or small sites. For this tutorial, you will be using the demo workspace that is automatically generated when you create a new account. Make sure you add an API key to this workspace in the API key settings. You will need your workspace ID and the (read-only) API key shortly.
```

Several of my suggestions here involve consistent [use of "you" as the pronoun](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun).